modules/serverless-rn-1-3-0.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-3-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.3.0
```

Comment:
One last general bit: We're dropping support for OCP 4.1. That'll ship with https://github.com/openshift-knative/serverless-operator/pull/57 and is the last thing blocking the release currently.